test-files/io/OsmChangesetElementTest/DeleteSplit2.osc

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<osmChange version="0.6" generator="hootenanny">
+</osmChange>
```

Comment:
This is the important one.  Here the second changeset is empty because I force the first to not be done when I create this second one.  Since there are no nodes that can be deleted yet (their way is being deleted in the first changeset) it comes out empty.